patches/lua-tree.patch

```diff
@@ -1099,3 +1074,318 @@ index 0121ba715..fc27a9b5e 100644
    requires = function(self)
      local conf_r = kong and
 
+diff --git a/lua-tree/share/lua/5.1/socket/url.lua b/lua-tree/share/lua/5.1/socket/url.lua
```

Comment:
I'm not sure why this section causes the changed to be added instead of removed when running the patch command...